dags/veda_data_pipeline/veda_discover_pipeline.py

```diff
@@ -13,33 +13,55 @@
 - This DAG can run with the following configuration <br>
 ```json
 {
-    "collection": "geoglam",
-    "prefix": "geoglam/",
+    "collection": "collection-id",
     "bucket": "veda-data-store-staging",
+    "prefix": "s3-prefix/",
     "filename_regex": "^(.*).tif$",
-    "discovery": "s3",
+    "id_regex": ".*_(.*).tif$",
```

Comment:
This makes sense based on `s3_discovery.py`, but could we add something in the docstring or README to explain the difference between `filename_regex` and `id_regex`?

Reply:
Definitely. Also going to need (based on some testing data that @slesaad prepared) another argument to prefix IDs (or template them with w.e `id_regex` captures